Packs/Gatewatcher/Integrations/GCenterv102/GCenterv102.yml

```
@@ -0,0 +1,1119 @@
+category: Network Security
+commonfields:
+  id: GCenterv102
+  version: -1
+configuration:
+- display: GCenter IP address
+  name: ip
+  type: 0
+  required: true
+- display: GCenter API token
+  name: token
+  type: 0
+  required: false
+- display: GCenter username
+  displaypassword: GCenter password
+  name: credentials
+  type: 9
+  required: false
+- display: Check the TLS certificate
+  name: check_cert
+  type: 8
+  required: false
+description: This integration allows, via about twenty commands, to interact with the GCenter appliance via its API.
+display: GCenter v2.5.3.102
+name: GCenterv102
+script:
+  commands:
+  - name: gw-list-alerts
+    description: List all alerts
+    outputs:
+    - contextPath: GCenter.Alerts.List.sha256
```

Please change all the keys here to 'Alert' (in all the relevant contextPaths here).
```suggestion
    - contextPath: GCenter.Alert.List.sha256
```
Please update that also in the readme file.